features/machine/machine.feature

```diff
@@ -23,6 +23,13 @@ Feature: Machine features testing
     Given evaluation of `cluster_operator('machine-api').condition(type: 'Progressing')` is stored in the :co_progressing clipboard
     Then the expression should be true> cb.co_progressing["status"]=="False"
 
+  # @author zhsun@redhat.com
+  # @case_id OCP-37706
+  @smoke
+  Scenario: Baremetal clusteroperator should be disabled in any deployment that is not baremetal
+    Given evaluation of `cluster_operator('baremetal').condition(type: 'Disabled')` is stored in the :co_disabled clipboard
```

Comment:
Do we need admin privilege here ?

Reply:
I checked this need admin privilege. But it's strange why another similar case can be passed in our runner-v3 job with normal TEST_MODE.
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/231698/console

Reply:
Generally we require admin test cases tag with admin by add `ensure_admin_tagged` in the step definiation, but the step use `evaluation of xxx` which can not be judged whether it requires admin or not (and in most case it does not require admin, so it does not add `ensure_admin_tagged`).

normal test mode in Runner-v3 job only ensures it does not run test cases which has tag admin or destructive. As the test case OCP-22115 does not have the tag, it's executed. While at the same time, you have passed in admin_creds_spec, so the test case use pseudo admin and got passed (The issue here is that the test case actually needs admin, but it does not tag the test case with admin. While during execution, admin privilege is provided, so it got pass).

Reply:
Thanks for the clear clarification, have added @admin